src/shared/utils/hodis.ts

```diff
@@ -28,7 +28,8 @@ export async function extractYearFromUser(kthId: string): Promise<{
 		const yearTagWithDash = `${yearTag
 			.slice(0, 1)
 			.toUpperCase()}-${yearTag.slice(1)}`;
-		return { yearRole: yearTagWithDash, year: hodisUser.year };
+		const year = 2000 + parseInt(yearTag.slice(1));
+		return { yearRole: yearTagWithDash, year: year };
```

Comment:
```suggestion
		return { yearRole: yearTagWithDash, year };
```

Reply:
I think this is something me and Benjamin did on the hacker night when he taught me how the whole thing was set up, idk why it is part of this branch tbh I think it has already been merged

Reply:
Benjamin mentions this in the Swedish comment above ^^
I simply forgot to make a new branch

Reply:
Peak foreshadowing